tests/build.go

```diff
@@ -415,3 +416,26 @@ func testBuildMultiPlatformNotSupported(t *testing.T, sb integration.Sandbox) {
 	require.Error(t, err, string(out))
 	require.Contains(t, string(out), "Multi-platform build is not supported")
 }
+
+func testDockerHostGateway(t *testing.T, sb integration.Sandbox) {
+	dockerfile := []byte(`
+FROM alpine
+RUN apk add syslinux
+RUN gethostip host.docker.internal | tee /hostip.txt
```

Comment:
You can use `nslookup` (or `ping -c 1`) to avoid adding network access to install packages.

Reply:
`host.docker.internal` looks like something that could be builtin. A name that is more test-specific could be better.